sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/implementation/util/ModelHelper.java

```diff
@@ -191,6 +191,13 @@ public static ShareFileDownloadHeaders transformFileDownloadHeaders(HttpHeaders
         }
     }
 
+    public static String getETag(HttpHeaders headers) {
+        if (headers == null) {
+            return null;
+        }
```

Comment:
should we assume headers aren't null ?

Reply:
that seems like a fair assumption. Will remove.